docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,102 @@
+## Making changes
 
-Assignee - Mason
-Reviewer - Sarah
+### Creating a branch
+
+Create a branch from your forked repo for your code changes:
+
+\```
+git checkout master
+git pull upstream master
+git checkout -b my-branch-name
+\```
+
+We strongly recommend using the CLI as your primary interface with git. GUIs are useful for viewing diffs, creating branches and making quick commits. However they often do not correctly merge and sync working branches with master. Mistakes in this step are time consuming to fix.
+
+That said, some team members use [SourceTree](https://www.sourcetreeapp.com/), [GitHub app](https://desktop.github.com/) or the git integrations available in VS Code.
+
+We don't have an official branch naming policy. However some developers use the following to make their branches easier to find: `user/[alias]/your-fancy-branch-name`, ie `user/johndoe/fix-that-bug`.
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time.
+
+```
+cd fluentui
+yarn
+```
+
+Run `yarn start` and select your start up project.
+
+```
+@fluentui/public-docsite // starts the public v8 documentation site.
+@fluentui/public-docsite-v9 // starts the internal v9 documentation storybook site.
+```
+
+Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push --force// Pushes your changes to your forked branch.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase -i master // Tacks your commits onto the end of master. Force is necessary since rebase changes history.
+```
+
+Resolve any conflicts in your editor.
+
+`git push upstream --force` // Pushes your changes to your forked branch. Include `upstream` if you have an open pull request, otherwise do not include it.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn lage build --since master` - build everything. It is good the run this weekly, or anytime you start a new project off master.
```

Comment:
```suggestion
- `yarn lage build --since master` - runs `build` task for all projects that are affected by your changes.
```

" anytime you start a new project off master"  -> TBH I dont think you should be running this command for regular dev flow on local, what folks should use is to run  `yarn buildto package-name` ( note: buildto will be removed after migration to nx )

Reply:
Updated :-)